spec.html

```diff
@@ -32412,6 +32562,101 @@ <h1>Expanded Years</h1>
           </emu-note>
         </emu-clause>
       </emu-clause>
+
+      <emu-clause id="sec-time-zone-offset-strings">
+        <h1>Time Zone Offset String Format</h1>
+
+        <p>
+          ECMAScript defines a string interchange format for the UTC offset of a time zone.
+          The format is described by the following grammar:
+        </p>
+
+        <emu-grammar type="definition">
+          ASCIISign ::: one of
```

Comment:
For readability, ECMAScript grammars are generally presented outside-in (i.e., from the outermost nonterminal, which here would seem to be |UTCOffsetString|).